src/components/views/rooms/MessageComposer.tsx

```diff
@@ -333,8 +332,12 @@ export class MessageComposer extends React.Component<IProps, IState> {
         this.messageComposerInput.current?.sendMessage();
 
         if (this.state.isWysiwygLabEnabled) {
+            // due to this function using wysiwyg functions targetting ES6,
+            // we need to import dynamically
+            const { sendMessage } = await import("./wysiwyg_composer/");
```

Comment:
I think that will be a good idea to wrap this dynamic import in the RTE files.

You can take a look at the `SendWysiwygComposer` composer for example